src/Sound/Tidal/Config.hs

```diff
@@ -25,31 +24,21 @@ data Config = Config {cCtrlListen :: Bool,
                       cCtrlAddr :: String,
                       cCtrlPort :: Int,
                       cCtrlBroadcast :: Bool,
-                      cFrameTimespan :: Double,
-                      cEnableLink :: Bool,
-                      cProcessAhead :: Double,
                       cTempoAddr :: String,
                       cTempoPort :: Int,
                       cTempoClientPort :: Int,
```

Comment:
Maybe this PR would be a good place to get rid of these, since it's already changing the `Config` type, and they're for a tempo-sharing mechanism that was replaced with Link?